docs/structured-streaming-programming-guide.md

```diff
@@ -1958,6 +1958,11 @@ Here are the configs regarding to RocksDB instance of the state store provider:
     <td>The waiting time in millisecond for acquiring lock in the load operation for RocksDB instance.</td>
     <td>60000</td>
   </tr>
+  <tr>
+    <td>spark.sql.streaming.stateStore.rocksdb.maxOpenFiles</td>
```

Comment:
I think eventually we want to have a JVM-wide level of config since the actual problem is not only from a single RocksDB instance in the executor but also from multiple RocksDB instances in the executor.

This config still requires end users to predict the number of stateful tasks being assigned and run in a single executor, which is exactly what we claimed that this is one of the issue in HDFS backed state store (due to memory) and we resolved the issue. But it is still a great start and lot better than what we have nothing.

Reply:
Sounds good @HeartSaVioR